src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -80,45 +84,67 @@ export class ContentTypeFieldsDropZoneComponent extends BaseComponent implements
      * @memberof ContentTypeFieldsDropZoneComponent
      */
     saveFieldsHandler(fieldToSave: Field): void {
-        const fields = this.getFields();
-        this.updateCurrentField(fieldToSave, fields);
+        const fields = this.getFieldsToSave(fieldToSave);
         this.emitSaveFields(fields);
         this.toggleDialog();
     }
 
+    private moveFields(): void {
+        const fields = this.getFields().filter((field, index) => {
+            const currentSortOrder = index + 1;
+
+            if (field.sortOrder !== currentSortOrder) {
```

Comment:
```
if (field.sortOrder !== currentSortOrder) {
    field.sortOrder = currentSortOrder;
}
return field.sortOrder !== currentSortOrder;
```

Reply:
this not work because after this: field.sortOrder = currentSortOrder 
the return value will be false

Reply:
True that.